streams/src/test/java/org/apache/kafka/test/InternalMockProcessorContext.java

```diff
@@ -193,6 +194,11 @@ public InternalMockProcessorContext(final File stateDir,
         this.metrics().setRocksDBMetricsRecordingTrigger(new RocksDBMetricsRecordingTrigger(new SystemTime()));
     }
 
+    @Override
+    protected StateManagerStub stateManager() {
```

Comment:
Should this be:

```suggestion
    protected StateManager stateManager() {
```

?